test/parallel/test-cluster-disconnect.js

```diff
@@ -72,7 +73,11 @@ if (cluster.isWorker) {
     let online = 0;
 
     for (let i = 0, l = workers; i < l; i++) {
-      cluster.fork().on('listening', common.mustCall(() => {
+      cluster.fork().on('listening', common.mustCall((address) => {
+        if (server_ports.indexOf(address.port) === -1) {
```

Comment:
IMHO can land as is (found 433 hits to `indexOf` in `/test/*.js`)
@lpinca since v4 entered maintenance, do you think we can eliminate all 433 now (in a different PR)?